static/style.css

```diff
@@ -248,4 +249,8 @@ kbd {
         border: 1px solid #eff1f3;
         box-shadow: inset 0 -1px 0 #eff1f3;
     }
+
+    mark {
+        color: #1f2328;
```

Comment:
Can we set background-color: #ffffaf for light?

or could be written as rgba(..., 1.0) as well but this doesn't really matter